infra/Freyfile.hcl

```diff
@@ -161,9 +178,8 @@ setup {
         }
       }
     }
-    tasks {
-      name     = "uppy-server | Set hostname"
-      hostname = "name={{lookup('env', 'FREY_DOMAIN')}}"
+    roles {
+      role = "{{{init.paths.roles_dir}}}/fqdn/1.2.1"
```

Comment:
I should not break since it's defined [here](https://github.com/transloadit/uppy-server/blob/master/infra/group_vars/all/_frey.yml#L25) now, but I can give it another test run if you want to :-)